libs/utils/utils.js

```diff
@@ -109,7 +110,9 @@ const AUTO_BLOCKS = [
   { 'pdf-viewer': '.pdf', styles: false },
   { video: '.mp4' },
   { merch: '/tools/ost?' },
-  { 'mas-autoblock': 'mas.adobe.com/studio' },
+  { 'mas-autoblock': 'mas.adobe.com/studio', styles: false },
```

Comment:
Since `styles` is set to `false`, is the `libs/blocks/mas-autoblock/mas-autoblock.css` empty file even needed?

Reply:
We used to have empty file `libs/blocks/mas-autoblock/mas-autoblock.css` but I removed it in this PR. That's why I set `styles: false` here

Reply:
Is it deleted? I still see it at https://github.com/adobecom/milo/blob/d59d2ed9ebe9f9e202568eeb8fc94cfe4660fb7e/libs/blocks/mas-autoblock/mas-autoblock.css 🤔